src/indexing_sources/source_sql.h

```diff
@@ -149,6 +149,8 @@ struct CSphSource_SQL : CSphSource_Document
 	virtual DWORD			SqlColumnLength ( int iIndex ) = 0;
 	virtual const char *	SqlColumn ( int iIndex ) = 0;
 	virtual const char *	SqlFieldName ( int iIndex ) = 0;
+	virtual const char *	SqlColumnStream ( int iIndex, DWORD &iStreamLength ) { iStreamLength = SqlColumnLength( iIndex ); return SqlColumn( iIndex ); }
+	virtual void			SqlColumnFreeStream ( char */*sStream*/ ) { }
```

Comment:
seems, that more generic `Release` is more appropriate than `Free` as part of the name (as generic - sometimes it is 'free', sometimes is just 'release', so it seems that more generic is better)